Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "NearestNeighbors"
 uuid = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
-version = "0.4.4"
+version = "0.5.0"
```

Comment:
Can this be 0.4.5?

Reply:
I'm not sure. This adds the "new feature" that you can use weighted Minkowski metrics in KDTrees. Also, in view of the Distances compat requirement, this is not "backward compatible", since (due to the inclusion of the weighted metrics), this requires the `Distances.parameters` function, which is available for weighted metrics only since v0.9.

Having said that, we could have a simple Distances bump with version 0.4.5 first, and then add the new feature at v0.5, in case that makes any noticeable difference.

Reply:
> This adds the "new feature" that you can use weighted Minkowski metrics in KDTrees.

Yeah, for pre 1.0 new features still go into patch releases.

> which is available for weighted metrics only since v0.9.

Yeah, but the compat changed so if you NN 0.4.5 you are guaranteed to have Distances 0.9 t?

Reply:
I have no preference regarding the version number. I'll change that to 0.4.5.